src/mongocxx/options/encrypt.cpp

```diff
@@ -12,16 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include <mongocxx/options/encrypt.hpp>
-
 #include <bsoncxx/types/private/convert.hh>
+#include <mongocxx/config/private/prelude.hh>
 #include <mongocxx/exception/error_code.hpp>
 #include <mongocxx/exception/logic_error.hpp>
+#include <mongocxx/options/encrypt.hpp>
 #include <mongocxx/private/libbson.hh>
 #include <mongocxx/private/libmongoc.hh>
 
-#include <mongocxx/config/private/prelude.hh>
```

Comment:
I suggest leaving the prelude header include here per CONTRIBUTING.md's guidelines.

Reply:
Done